src/styles/tailwind.css

```diff
@@ -0,0 +1,1101 @@
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
+
+@layer base {
+  :root {
+    --background: 0 0% 100%;
```

Comment:
to make UI library compatible with obsidian plugin, we should use the CSS variables defined in obsidian (https://docs.obsidian.md/Reference/CSS+variables/CSS+variables). Many obsidian users use themes, which are made by altering those CSS variables (https://docs.obsidian.md/Themes/App+themes/Build+a+theme). Users would expect a well-designed plugin to change the appearance accordingly.

This is the main reason why I believe shadcn is the right choice for obsidian plugin UI library. It provides developer great flexibility to change how components are styled. We will lose the value if we use the shadcn default CSS variables.

I suggest:
1. update tailwind.config.js to create tokens that map to obsidian CSS variables, including color, spacing, sizing, etc
2. whenever we introduce a shadcn component, restyle it with the new token before we integrate it into the plugin.

Reply:
@zeroliu sometimes, I don't want to use some of the built-in theme styles.`(eg: radius)`
so, I think we should select some css styles to map.